src/components/pages/gallery/Upload.tsx

```diff
@@ -121,11 +156,29 @@ export default function Upload(props: Props) {
         setProgressView(true);
     };
 
+    const resumeDesktopUpload = async (
+        files: ElectronFile[],
+        collectionName: string
+    ) => {
+        try {
+            if (files && files?.length > 0) {
+                isPendingDesktopUpload.current = true;
+                pendingDesktopUploadCollectionName.current = collectionName;
+                props.setElectronFiles(files);
+            }
+        } catch (e) {
+            logError(e, 'Failed to get previously failed files');
```

Comment:
nit: review and check if we need to update the error message? What we are doing inside method doesn't seem to be matching the the error copy. 
Also, I see that we are swallowing the error here. Always try to document the reason for such cases :). It will help future maintainer and they won't think it as a mistake.

Reply:
I removed the try-catch since the functionality had changed a bit and I don't think we need it now.

And yes will keep in mind to document any such things. 🙂